src/main/java/dk/sdu/se_f22/sortingmodule/infrastructure/domain/SortingModuleImpl.java

```diff
@@ -61,6 +62,8 @@ public void setScoring(String scoring) {
 
     @Override
     public SearchHits search() {
+        SearchModuleImpl seachModule = new SearchModuleImpl(this.searchString);
```

Comment:
Error from search module... Lovely

Reply:
I saw it my self. just corrected it. Test passed.